index.html

```diff
@@ -200,46 +219,59 @@ <h2 class="about-me">
-            <div class="container-duo">
-                <h2 class="item-rotated-square">  </h2>
-                <h2 class="item-2"> Language </h2>
-            </div>
-    
-            <ul class="list-arrange">
-                <li class="language-item">Java Script</li>
-                <li class="language-item">Ruby</li>
-                <li class="language-item">Html</li>
-                <li class="language-item">Css</li>
-            </ul>        
-
-            <div class="container-duo">
-                <h2 class="item-square">  </h2>
-                <h2 class="item-2"> Frameworks </h2>
+
+            <div class="wrapper-item-about">
+                <div class="container-duo">
+                    <h2 class="item-rotated-square">  </h2>
+                    <h2 class="item-2"> Language </h2>
+                </div>
+        
+                <ul class="list-arrange">
+                    <li class="language-item">Java Script</li>
+                    <li class="language-item">Ruby</li>
+                    <li class="language-item">Html</li>
+                    <li class="language-item">Css</li>
+                </ul>                
+            </div> 
+            
+            
+            <div class="wrapper-item-about">
+                <div class="container-duo">
+                    <h2 class="item-square">  </h2>
+                    <h2 class="item-2"> Frameworks </h2>
+                </div>
+        
+                <ul class="list-arrange">
+                    <li class="language-item">Bootstrap</li>
+                    <li class="language-item">Ruby on Rails</li>
+                    <li class="language-item">RSPec</li>
+                    <li class="language-item">CapyBara</li>
+                    <li class="language-item">Selenium</li>            
+                </ul>
             </div>
-    
-            <ul class="list-arrange">
-                <li class="language-item">Bootstrap</li>
-                <li class="language-item">Ruby on Rails</li>
-                <li class="language-item">RSPec</li>
-                <li class="language-item">CapyBara</li>
-                <li class="language-item">Selenium</li>            
-            </ul>
-    
-            <div class="container-duo">
-                <h2 class="item-circle"></h2>
-                <h2 class="item-2">Skills</h2>
+
+
+            <div class="wrapper-item-about">
+                <div class="container-duo">
+                    <h2 class="item-circle"></h2>
+                    <h2 class="item-2">Skills</h2>
+                </div>
+        
+                <ul class="list-arrange">
+                    <li class="language-item">Codekit</li>
+                    <li class="language-item">Github</li>
+                    <li class="language-item">Codepen</li>
+                    <li class="language-item">Gitlab</li>
+                    <li class="language-item">Terminal</li>            
+                </ul>
```

Comment:
[OPTIONAL] this is meant to be required but since it was missed by the previous reviewers I'm tagging it as optional. Please fix this overflow problem which hides away one of the list.


<img width="650" alt="optional" src="https://user-images.githubusercontent.com/84930355/206860148-a75f9402-42dd-4cbb-9f5a-a674ffc9ba1d.png">

Reply:
It is strange, are you sure you are running the last version? cause it runs ok on my computer.